torch_np/_dtypes.py

```diff
@@ -183,6 +181,23 @@ def is_integer(dtyp):
     return dtyp.typecode in typecodes['AllInteger']
 
 
+
+def issubclass_(arg, klass):
```

Comment:
This goes on my list of functions to deprecate and remove in NumPy 2.0 ...

Reply:
This one is easy enough so may as well leave it here now. I'd say let's not bother with `issubsctype`, `obj2sctype`, etc., those definitely all need to go from NumPy.